README.md

```diff
@@ -1 +1,19 @@
-# execute-watcher
\ No newline at end of file
+# Notify watcher core
+
+Utilities for running Notify watcher watcher's
+
+## Usage
+
+Run a watcher by using `execute`
+
+```js
+const { execute } = require('@notify-watcher/core');
+
+// Replace with your watcher
+const watcher = {}
+
+// Provided required data to watcher
```

Comment:
Maybe this line can be explained a bit more

Reply:
And these same options are then passed on to the same watchers you are giving here. I don't understand this part

Reply:
The server uses `execute` to call the watchers